docs/conf.py

```diff
@@ -53,8 +54,10 @@
 modindex_common_prefix = ["template_project."]
 
 # html theme options
-html_static_path = ["_static"]
-html_logo = "_static/images/logo.png"
+html_title = f"{project} {release}"
+# html_logo = "_static/images/logo.png"
```

Comment:
Better to delete this, imo. Commented code is confusing to people why it was commented out and if it is safe to remove or not.